var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -60,93 +74,82 @@ class Gcc(Package):
     patch('piclibs.patch', when='+piclibs')
     patch('gcc-backport.patch', when='@4.7:5.3')
 
-    def install(self, spec, prefix):
-        # libjava/configure needs a minor fix to install into spack paths.
-        filter_file(r"'@.*@'", "'@[[:alnum:]]*@'", 'libjava/configure',
-                    string=True)
```

Comment:
I removed this thing. I'm pretty sure this was added back in the day when the Spack installation directories still included special characters like `@`, but I don't think this is necessary anymore.